### PR TITLE
Fixed a bug when used `consumer_update_listener` callback from another subscription

### PR DESCRIPTION
**Pull request by nesb1**
Hi
Fixed a bug when used `consumer_update_listener` callback from another subscription in single active consumer mode.

### CODE REVIEW

File: tests/test_consumer.py [hunk 1]
```diff
@@ -400,6 +400,71 @@ async def consumer_update_listener_with_custom_offset(
...
 
+async def test_consume_with_multiple_sac_custom_consumer_update_listener_cb(
+    consumer: Consumer, producer: Producer
+) -> None:
+    stream_name_1 = "stream1"
+    stream_name_2 = "stream2"
+    await producer.create_stream(stream=stream_name_1)
+    await producer.create_stream(stream=stream_name_2)
+    try:
+        # necessary to use send_batch, since in this case, upon delivery, rabbitmq will deliver
```
Comment (nesb1) [inline tests/test_consumer.py:411]: Invalid comment

Comment (nesb1) [reply tests/test_consumer.py]: fixed

**Review by DanielePalaia — APPROVED** [review]
LGTM! Thanks

**Comment by nesb1** [comment]
@DanielePalaia @Gsantomaggio Hi, these changes merged, but not released. Can you release it?

**Comment by Gsantomaggio** [comment]
Hi @nesb1 
Thank you for the remainder. We just released a new version.

**Pull request merged**